src/ProjNavigator/sources_form.cpp

```diff
@@ -280,9 +281,15 @@ void SourcesForm::SlotRemoveFile() {
   QTreeWidgetItem *item = m_treeSrcHierachy->currentItem();
   if (item == nullptr) return;
 
+  auto strFileName = item->text(0);
+  if (strFileName.contains(SRC_TREE_FLG_TOP))
+    strFileName.remove(SRC_TREE_FLG_TOP);
   auto fileSet = item->data(0, SetFileDataRole);
   if (!fileSet.isNull()) {
-    QString strFileName = item->text(0);
+    auto questionStr = tr("Should %1 file be removed?").arg(strFileName);
+    if (QMessageBox::question(this, tr("Remove file"), questionStr) ==
```

Comment:
Might be able to clarify the remove action by changed the message box title to "Remove file from project?"

Reply:
Do we have a general practice to add titles to messageboxes ?

Reply:
Could you elaborate on your question? Not sure what is being asked here. QMessageBox always has a title, it's set in line 290 by tr("Remove file").

Reply:
I went through message boxes used in our application. They mainly have a title, but not always:
WidgetFactory.cpp:
QMessageBox confirm(parent);
confirm.setWindowTitle("Save Changes?");

main_window.cpp:
auto answer = QMessageBox::question(
        this, "Save constraint file...", ...
        
On the other hand, text_editor_form.cpp has 'titleless' message box (but that might be the only one):
int ret = QMessageBox::question(
        this, tr(""), tr("Save changes in %1?"

Reply:
Ok, thanks for sharing that. Let's remove the title in this case and update the sentence as mentioned in comment https://github.com/os-fpga/FOEDAG/pull/647#discussion_r971105075